core/chains/evm/client/errors.go

```diff
@@ -465,3 +465,19 @@ func NewSendErrorReturnCode(err error, lggr logger.Logger, tx *types.Transaction
 	}
 	return clienttypes.Unknown, err
 }
+
+// Solution to handle SendOnly nodes error codes. In that case, we don't assume there is another transaction that will be correctly
+// priced. Ideally in the future we wouldn't need an entire new function for this.
+func NewSendOnlyErrorReturnCode(err error, lggr logger.Logger, tx *types.Transaction, fromAddress common.Address, isL2 bool) (clienttypes.SendTxReturnCode, error) {
+	sendError := NewSendError(err)
+	if sendError.IsNonceTooLowError() || sendError.IsTransactionAlreadyMined() {
+		// Nonce too low indicated that a transaction at this nonce was confirmed already.
+		// Mark it as TransactionAlreadyKnown.
+		return clienttypes.TransactionAlreadyKnown, err
+	}
+	if sendError.IsTransactionAlreadyInMempool() {
+		lggr.Debugw("Transaction already in mempool", "txHash", tx.Hash, "nodeErr", sendError.Error())
+		return clienttypes.Successful, err
```

Comment:
We actually don't even need this function anymore since the multiNodeClient has been simplified and no longer handles SendTransaction requests. I can revert the functionality to how it worked previously for EVM now